### PR TITLE
Fixed azure artifacts path download.

### PR DESCRIPTION
**Pull request by ipanova**
closes #8427
https://pulp.plan.io/issues/8427

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

File: pulpcore/content/handler.py [hunk 1]
```diff
@@ -594,15 +594,16 @@ def _serve_content_artifact(self, content_artifact, headers):
...
-            settings.DEFAULT_FILE_STORAGE == "storages.backends.s3boto3.S3Boto3Storage"
-            or settings.DEFAULT_FILE_STORAGE == "storages.backends.azure_storage.AzureStorage"
-        ):
+        elif settings.DEFAULT_FILE_STORAGE == "storages.backends.s3boto3.S3Boto3Storage":
             artifact_file = content_artifact.artifact.file
             content_disposition = f"attachment;filename={content_artifact.relative_path}"
             parameters = {"ResponseContentDisposition": content_disposition}
             url = artifact_file.storage.url(artifact_file.name, parameters=parameters)
             raise HTTPFound(url)
+        elif settings.DEFAULT_FILE_STORAGE == "storages.backends.azure_storage.AzureStorage":
```
Comment (ipanova) [inline pulpcore/content/handler.py:603]: @daviddavis should not we also set content-disposition header for Azure artifacts? 
https://docs.microsoft.com/en-us/rest/api/storageservices/set-blob-properties#request-headers-all-blob-types

Comment (daviddavis) [reply pulpcore/content/handler.py:603]: I would think so but I am not familiar with Azure and I haven't tested with it at all.

Comment (ipanova) [reply pulpcore/content/handler.py:603]: I agree, I am not familiar with Azure either. I wanted to finalize this community contribution. Can we merge this as-is and add/fix content-disposition support later on whenever we receive a bug?Thoughts?

**Review by daviddavis — APPROVED** [review]

**Review by goosemania — APPROVED** [review]
Thanks!

**Pull request merged**